content/projects/understanding-loops/_index.md

```diff
@@ -77,24 +77,54 @@ Please note that one person can have multiple baskets. If you look at the data,
 
 ## Write some functionality
 
+### Important notes for Javascript solutions
+
+In Javascript it is best practice to use destructuring when passing arguments to a function. You can learn about destructuring here: [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment).
+
+Please make sure you use destructuring for all your functions. 
+
+```
+getCustomerBaskets(shoppingBaskets, "sine@umuzi.org"); // DON'T DO THIS
+getCustomerBaskets("sine@umuzi.org", customerBaskets); // DON'T DO THIS EITHER
+
+// The problem with the above code is that if you get the order of the parameters wrong, then things will break. Your functions should work like this instead:
+
+function getCustomerBaskets({ shoppingBaskets, email }) {...}
+
+getCustomerBaskets({shoppingBaskets, email}) //DO THIS
+``` 
+
 ### get baskets belonging to a single customer
 
 Write a function called `get customer baskets` that takes in the email address and the data array as arguments and returns a list/array of all the shopping baskets that belong to the customer with that email address.
 
 If the customer has no shopping baskets then return an empty list/array.
 
-e.g. `getCustomerBaskets("tshepo@umuzi.org", dataStore);`
+e.g. 
+```
+getCustomerBaskets({ email, shoppingBaskets }); //javascript
+get_customer_baskets(email, shopping_baskets) // python 
```

Comment:
Please update for java as well. It should look like JS, but without the destructured arguments